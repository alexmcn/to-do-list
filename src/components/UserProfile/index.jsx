import React from 'react';
import PropTypes from 'prop-types';

import { makeStyles } from '@material-ui/core/styles';

import { Avatar, Grid, Typography } from '@material-ui/core';

import ProfileImage from './profile-image.png';

const useStyles = makeStyles({
  avatar: {
    margin: 10,
    width: 60,
    height: 60,
  },
});

const UserProfile = ({ userName }) => {
  const classes = useStyles();

  return (
    <Grid container alignItems="center">
      <Avatar alt={userName} src={ProfileImage} className={classes.avatar} />
      <Typography />
    </Grid>
  );
};

UserProfile.propTypes = {
  userName: PropTypes.string,
};

export default UserProfile;
