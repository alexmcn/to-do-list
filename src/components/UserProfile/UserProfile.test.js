import React from 'react';
import renderer from 'react-test-renderer';

import UserProfile from './index';

describe('UserProfile', () => {
  it('has a profile pic and username', () => {
    const instance = renderer.create(<UserProfile userName="JohnSmith" />);

    expect(instance.toJSON()).toMatchSnapshot();
  });
});
