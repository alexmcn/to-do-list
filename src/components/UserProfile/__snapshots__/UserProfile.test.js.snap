// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile has a profile pic and username 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
>
  <div
    className="MuiAvatar-root makeStyles-avatar-1"
  >
    <img
      alt="JohnSmith"
      className="MuiAvatar-img"
      src="profile-image.png"
    />
  </div>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    JohnSmith
  </p>
</div>
`;
